purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/events/CustomerCenterEventType.kt

```diff
@@ -0,0 +1,23 @@
+package com.revenuecat.purchases.customercenter.events
+
+/**
+ * Types of events that can be tracked in the Customer Center. Meant for RevenueCatUI use.
+ */
+enum class CustomerCenterEventType(val value: String) {
+    /**
+     * The customer center was shown to the user.
+     */
+    IMPRESSION("customer_center_impression"),
+
+    /**
+     * The customer center was closed by the user.
+     */
+    SURVEY_OPTION_CHOSEN("customer_center_survey_option_chosen"), // Fixed typo
+}
+
+/**
+ * Display mode for the Customer Center. Meant for RevenueCatUI use.
+ */
+enum class CustomerCenterDisplayMode {
```

Comment:
Hmm a couple questions here... Are we planning to support other display modes in the short/medium term? If not, maybe we can avoid adding this for now?

In case we do, I wonder if this would be better in a separate file? Seems weird that it's within the event type when it shouldn't really have anything to do with events. Also, same about the `@ExperimentalPreviewRevenueCatPurchasesAPI` annotation

Reply:
We don't. As far as I understand, there's no other display mode in Android. In iOS we do have two. I thought it would be nice to have them listed anyway, just for consistency between platforms; but I can remove it 👌

Reply:
I'll keep it for now, but we can come back to this once the whole event structure is set up